modules/kube-loadbalancer-sg/main.tf

```diff
@@ -0,0 +1,26 @@
+/**
+ * ## Kube Load Balancer Security Group
+ *
+ * Define a security group for kube load balancer
+ *
+ */
+
+module "kube-load-balancer-sg" {
+  source      = "../security-group-base"
+  name        = "${var.name}"
+  vpc_id      = "${module.vpc.vpc_id}"
+  description = "Security group for the kube load-balancer in ${var.name}"
+}
+
+module "kube-load-balancer-api-rule" {
```

Comment:
let's rename these to `api-rule` and similar, to drop the `kube-load-balancer`. This change will make the names in the Terraform state a little shorter and easier to track.